core-bundle/src/Resources/contao/widgets/PageSelector.php

```diff
@@ -239,7 +239,7 @@ public function generate()
 
 		// Return the tree
 		return '<ul class="tl_listing tree_view picker_selector' . ($this->strClass ? ' ' . $this->strClass : '') . '" id="' . $this->strId . '" data-callback="reloadPagetree" data-inserttag="link_url">
-    <li class="tl_folder_top"><div class="tl_left">' . Image::getHtml($GLOBALS['TL_DCA']['tl_page']['list']['sorting']['icon'] ?: 'pagemounts.svg') . ' ' . $GLOBALS['TL_LANG']['MOD']['page'][0] . '</div> <div class="tl_right">&nbsp;</div><div style="clear:both"></div></li><li class="parent" id="' . $this->strId . '_parent"><ul>' . $tree . $strReset . '
+    <li class="tl_folder_top"><div class="tl_left">' . Image::getHtml($GLOBALS['TL_DCA']['tl_page']['list']['sorting']['icon'] ?? 'pagemounts.svg') . ' ' . ($GLOBALS['TL_LANG']['MOD']['page'][0] ?? '') . '</div> <div class="tl_right">&nbsp;</div><div style="clear:both"></div></li><li class="parent" id="' . $this->strId . '_parent"><ul>' . $tree . $strReset . '
```

Comment:
How can `$GLOBALS['TL_LANG']['MOD']['page'][0]` ever be not set?

Reply:
if the `modules.php` language file is not loaded 🤷